lessonbuilder/tool/src/webapp/css/Simplepagetool.css

```diff
@@ -1385,7 +1388,7 @@ div[role="dialog"] {
     background: url(../images/reload_h.gif) center left no-repeat
 }
 
-.right-col {
+div.Mrphs-portalWrapper .right-col {
```

Comment:
I'd move this up to the other column stuff and again, comment about you're wanting to detect when it's in the portal and when it's not.